### PR TITLE
Format MapRenderer according to project code style

### PR DESCRIPTION
**Pull request by hartez**
### Description of Change ###

Just doing the reformat of Android's MapRenderer.cs that somehow got missed when we standardized everything else.

### CODE REVIEW

File: Xamarin.Forms.Maps.Android/MapRenderer.cs [hunk 1]
```diff
@@ -10,311 +11,358 @@
...
+				return;
+			}
+			if (_markers == null)
+			{
+				return;
+			}
+
+			foreach (Pin p in pins)
+			{
+				Marker marker = _markers.FirstOrDefault(m => (object)m.Id == p.Id);
```
Comment (rmarinho) [inline Xamarin.Forms.Maps.Android/MapRenderer.cs]: why not var ?

File: Xamarin.Forms.Maps.Android/MapRenderer.cs [hunk 2]
```diff
@@ -10,311 +11,358 @@
...
-			var marker = eventArgs.Marker;
+			Marker marker = eventArgs.Marker;
 
 			// lookup pin
 			Pin targetPin = null;
-			for (var i = 0; i < Map.Pins.Count; i++) {
-				var pin = Map.Pins[i];
+			for (var i = 0; i < Map.Pins.Count; i++)
+			{
+				Pin pin = Map.Pins[i];
```
Comment (rmarinho) [inline Xamarin.Forms.Maps.Android/MapRenderer.cs]: why change from var?

File: Xamarin.Forms.Maps.Android/MapRenderer.cs [hunk 3]
```diff
@@ -10,311 +11,358 @@
...
-			var map = NativeMap;
+			GoogleMap map = NativeMap;
 			if (map == null)
+			{
 				return;
+			}
 
 			if (_markers == null)
-				_markers = new List<Marker> ();
+			{
```
Comment (rmarinho) [inline Xamarin.Forms.Maps.Android/MapRenderer.cs]: we normally drop if we have 1 line that's not inside other {}

File: Xamarin.Forms.Maps.Android/MapRenderer.cs [hunk 4]
```diff
@@ -10,311 +11,358 @@
...
-			var gmap = NativeMap;
+
+			GoogleMap gmap = NativeMap;
 			if (gmap == null)
+			{
 				return;
+			}
 
 			if (e.PropertyName == Map.IsShowingUserProperty.PropertyName)
+			{
```
Comment (rmarinho) [inline Xamarin.Forms.Maps.Android/MapRenderer.cs]: it was ok without this.. i prefer the old way.

File: Xamarin.Forms.Maps.Android/MapRenderer.cs [hunk 5]
```diff
@@ -10,311 +11,358 @@
...
-			AutoPackage = false;
-		}
+		const string MoveMessageName = "MapMoveToRegion";
 
 		static Bundle s_bundle;
-		internal static Bundle Bundle { set { s_bundle = value; } }
+
+		bool _disposed;
+
+		bool _init = true;
```
Comment (rmarinho) [inline Xamarin.Forms.Maps.Android/MapRenderer.cs]: strange that this is true to starts

Comment (hartez) [inline Xamarin.Forms.Maps.Android/MapRenderer.cs]: This PR is just to get the formatting in line with everything else; we're not worried about logic issues here.

File: Xamarin.Forms.Maps.Android/MapRenderer.cs (hunk 4 above)
Comment (hartez) [inline Xamarin.Forms.Maps.Android/MapRenderer.cs]: For this particular case, I think it looks weird to have an `if...else if...else if` where some of the cases are in braces and some aren't; the last case has two statements, so it needs braces. Adding them to the rest of the cases makes it consistent.

Comment (hartez) [inline Xamarin.Forms.Maps.Android/MapRenderer.cs]: ... and, looking at the our [code style guidelines](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)  it looks like the corefx rule is that [if any block in the statement needs braces, they all do](https://github.com/dotnet/corefx/issues/381#issuecomment-69474542).

File: Xamarin.Forms.Maps.Android/MapRenderer.cs (hunk 2 above)
Comment (hartez) [inline Xamarin.Forms.Maps.Android/MapRenderer.cs]: Trying to follow [rule 10](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md), which ReSharper has a setting for. But ReSharper's decision here is odd; it's pretty obvious what the type is. I'm going to change that one back.

File: Xamarin.Forms.Maps.Android/MapRenderer.cs (hunk 1 above)
Comment (hartez) [inline Xamarin.Forms.Maps.Android/MapRenderer.cs]: No reason it couldn't be, I suppose. The fact that it's coming from an enumerable called `_markers` makes the type obvious enough to pass [rule 10](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md).

File: Xamarin.Forms.Maps.Android/MapRenderer.cs (hunk 4 above)
Comment (rmarinho) [inline Xamarin.Forms.Maps.Android/MapRenderer.cs]: 👍

**Pull request merged**